src/components/cards/LearnCard.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Column, Columns, Inline, Stack, Text } from '@/design-system';
+import React, { useCallback } from 'react';
+import { GenericCard } from './GenericCard';
+import { getLearnCardColorway } from './utils/constants';
+import { LearnCardDetails } from './utils/types';
+import { IconOrb } from './reusables/IconOrb';
+import { useNavigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+import * as i18n from '@/languages';
+import { useTheme } from '@/theme';
+import { useRoute } from '@react-navigation/native';
+import { analyticsV2 } from '@/analytics';
+
+export const LEARN_CARD_HEIGHT = 184;
+
+type LearnCardProps = {
+  cardDetails: LearnCardDetails;
+  type: 'square' | 'stretch';
+};
+
+export const LearnCard = ({ cardDetails, type }: LearnCardProps) => {
+  const { navigate } = useNavigation();
+  const { isDarkMode } = useTheme();
+  const { category, emoji, url, key } = cardDetails;
+  const { name: routeName } = useRoute();
+  const {
+    gradient,
+    shadowColor,
+    orbColorLight,
+    primaryTextColor,
+    secondaryTextColor,
+  } = getLearnCardColorway(category, isDarkMode);
+
+  const onPress = useCallback(() => {
+    analyticsV2.track(analyticsV2.event.cardPressed, {
+      cardName: 'LearnCard',
+      fromScreen: routeName,
+      cardType: type,
+    });
+    navigate(Routes.LEARN_WEB_VIEW_SCREEN, {
+      category,
+      url,
+      displayType: type,
+      fromScreen: routeName,
+      key,
+    });
+  }, [category, key, navigate, routeName, type, url]);
+
+  const translations = i18n.l.cards.learn;
+
+  return (
+    <GenericCard
+      type={type}
+      gradient={gradient}
+      onPress={onPress}
+      color={shadowColor}
+    >
+      {type === 'square' ? (
+        <Box height="full" justifyContent="space-between">
+          <Inline alignHorizontal="justify">
+            <Text
+              size="13pt"
+              weight="heavy"
+              color={{ custom: primaryTextColor }}
+            >
+              {`􀫸 ${i18n.t(translations.learn).toUpperCase()}`}
+            </Text>
+            <IconOrb color={orbColorLight} icon={emoji} />
+          </Inline>
+          <Stack space="10px">
+            <Text
+              color={{ custom: secondaryTextColor }}
+              size="13pt"
+              weight="bold"
+            >
+              {i18n.t(translations.categories[category])}
+            </Text>
+            <Text
+              color={{ custom: primaryTextColor }}
+              size="17pt"
+              weight="heavy"
+            >
+              {i18n.t(translations.cards[key].title)}
+            </Text>
+          </Stack>
+        </Box>
+      ) : (
+        <Stack space="36px">
+          <Box height={{ custom: 65 }}>
+            <Columns space="20px">
+              <Column>
+                <Stack space="12px">
```

Comment:
Again, lack of design system knowledge: why Stack and not just margin/padding?

Reply:
i remember i originally just implemented it w/ `Stack` and `Inline`, but there was a weird issue with text wrapping or something. I think when the title text grew large enough instead of wrapping appropriately it would instead extend beyond its container and mess with the alignment of the neighboring `IconOrb` component. not sure why using Columns fixed it